tests/auto/gltfparser/tst_gltfparser.cpp

```diff
@@ -855,6 +855,27 @@ private Q_SLOTS:
         // THEN
         QVERIFY(res != nullptr);
     }
+
+    void checkSparseAccessors()
+    {
```

Comment:
Would it make sense to test against a file that has a sparse accessor that doesn't reference a bufferview to ensure the 0 filled generation works?

Reply:
done

Reply:
done